socorro/processor/rules/breakpad.py

```diff
@@ -354,42 +255,29 @@ def action(self, raw_crash, raw_dumps, processed_crash, processor_meta):
 
                 stackwalker_data, return_code = self._execute_external_process(
                     crash_id,
+                    self.command_pathname,
                     command_line,
                     processor_meta
                 )
 
-                if dump_name == self.config.dump_field:
+                if dump_name == self.dump_field:
                     processed_crash.update(stackwalker_data)
                 else:
                     processed_crash.additional_minidumps.append(dump_name)
                     processed_crash[dump_name] = stackwalker_data
 
 
-class JitCrashCategorizeRule(ExternalProcessRule):
-    # FIXME(willkg): command_line and command_pathname are referenced in the
-    # uplifted versions in ProcessorPipeline. The rest of these config values have
-    # no effect on anything and are just here.
-    required_config = Namespace()
-    required_config.add_option(
-        'command_line',
-        doc='template for the command to invoke the external program; uses Python format syntax',
-        default='timeout -s KILL 30 {command_pathname} {dump_file_pathname} '
-    )
-    required_config.add_option(
-        'command_pathname',
-        doc='the full pathname to the external program to run (quote path with embedded spaces)',
-        default='/data/socorro/stackwalk/bin/jit-crash-categorize',
```

Comment:
Note that this is the old path from when we were running RPM-based Socorro in CentOS. The new path is `/stackwalk/jit-crash-categorize`.